src/NServiceBus.TransactionalSession/OpenSessionOptions.cs

```diff
@@ -0,0 +1,41 @@
+namespace NServiceBus.TransactionalSession
+{
+    using System;
+    using System.Collections.Generic;
+    using Extensibility;
+
+    /// <summary>
+    /// Allows the users to control how the transaction session behaves.
+    /// </summary>
+    /// <remarks>
+    /// The behavior of this class is exposed via extension methods.
+    /// </remarks>
+    public class OpenSessionOptions : IExtendable
```

Comment:
I feel `TransactionalSessionOptions` would be more aligned with the current "options naming scheme".